### PR TITLE
Word Filter

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
Adds a word filter

## Why It's Good For The Game
trying to make the game more RP oriented because gamer words are very LRP

## Changelog
:cl:
add: Added word filter
/:cl:

### CODE REVIEW

**Comment by coldud13** [comment]
based

File: strings/gamer_words.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+owu
+brb
+afk
+omg
+omfg
+rofl
+stfu
+nvm
+ily
+ttyl
```
Comment (coldud13) [inline strings/gamer_words.txt:24]: spot the boomer

File: code/modules/mob/say.dm [hunk 2]
```diff
@@ -20,6 +20,19 @@
...
 		return
+	//lumos
+	if(isliving(src))
+		var/mob/living/LivingMob = src
+		var/list/check_gamer_words = splittext(message, " ")
+		for(var/checked_text in check_gamer_words)
+			var/lower_checked_text = lowertext(checked_text)
+			if(lower_checked_text in GLOB.gamer_words)
+				to_chat(LivingMob, "<span class='warning'>You feel embarrassed that you were going to say [lower_checked_text].</span>")
+				message_admins("[key_name(LivingMob)] tried to say [lower_checked_text].")
```
Comment (coldud13) [inline code/modules/mob/say.dm:31]: not too sure how i feel about this

**Comment by faaaay** [comment]
I'd have to disagree with _bruh_ being filtered. It's a little c r o n g e, but something that might plausibly be said.

**Comment by coldud13** [comment]
ill have to redo this to use regex later on

**Pull request merged**